src/App.js

```diff
@@ -22,12 +22,11 @@ const App = (props) => (
           render={() => (
```

Comment:
Consider storing this function ( () => Profile...) in a separate variable in the current function.

Reply:
/fxd